src/modm/platform/can/stm32-fdcan/can.cpp.in

```diff
@@ -377,7 +375,7 @@ modm::platform::Fdcan{{ id }}::getMessage(can::Message& message, uint8_t *filter
 			(*timestamp) = rxMessage.timestamp;
 		}
 		rxQueue.pop();
-		NVIC_EnableIRQ({{ reg }}_IT1_IRQn);
+		{{ reg }}->ILE = FDCAN_ILE_EINT1 && FDCAN_ILE_EINT0; // reenable ISR, in case it was disabled due to rxQueue full
```

Comment:
You probably mean `FDCAN_ILE_EINT1 | FDCAN_ILE_EINT0` :wink: . `FDCAN_ILE_EINT1 && FDCAN_ILE_EINT0` evaluates to `true` which is converted to `1` on assignment.

Reply:
😳🙈🤦‍♂️🤦‍♂️🤦‍♂️
Wow. That's silly

Reply:
Thanks